### PR TITLE
PRM-384 Added AppealStatus TRIBUNAL_REJECTED

### PR DESCRIPTION
**Pull request by ruthdas03**

### CODE REVIEW

File: app/viewmodels/SummaryCardHelper.scala [hunk 1]
```diff
@@ -289,6 +290,13 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
           link("#", "summaryCard.appeal.readMessageReinstated")
         )
       }
+      case AppealStatusEnum.Tribunal_Rejected => {
```
Comment (LewisMitchell) [inline app/viewmodels/SummaryCardHelper.scala:293]: I don't see a difference between this scenario and the Accepted | Rejected scenarios.
Couldn't you just add `| AppealStatusEnum.Tribunal_Rejected` to line 279? and remove this block?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good, just a minor change.
Could you add a test in the SummaryCardSpec? To check the html is correct, see:
https://github.com/hmrc/penalties-frontend/pull/42/files#diff-e4b1f9a64ca565a11f7a68c5630be5cdab6c9588cd7444128c423c36583f6db8R370-R372

File: app/viewmodels/SummaryCardHelper.scala (hunk 1 above)
Comment (ruthdas03) [reply app/viewmodels/SummaryCardHelper.scala]: done

**Review by LewisMitchell — APPROVED** [review]

**Comment by LewisMitchell** [comment]
test this please

**Comment by LewisMitchell** [comment]
image

**Pull request merged**